src/index.js

```diff
@@ -41,13 +41,13 @@ const EcomApps = function (ecomAuth = new EcomAuth()) {
    */
   this.ecomAuth = ecomAuth
 
-  this.fetchMarketApps = meta => fetchMarketApps(meta)
-  this.findApp = appId => findApp(appId)
-  this.installApp = (appId, redirect, body) => installApp(self, appId, redirect, body)
-  this.removeApplication = _id => removeApplication(self, _id)
-  this.editApplication = (_id, body, canUpdateInternals) => editApplication(self, _id, body, canUpdateInternals)
-  this.fetchStoreApplications = meta => fetchStoreApplications(self, meta)
-  this.findStoreApplication = _id => findStoreApplication(self, _id)
+  this.listFromMarket = meta => listFromMarket(meta)
+  this.findOnMarket = appId => findOnMarket(appId)
```

Comment:
Acho que é interessante convencionar que todos os métodos recebam `self` no primeiro parâmetro, só pra deixar mais previsível mesmo...

```suggestion
  this.listFromMarket = meta => listFromMarket(self, meta)
  this.findOnMarket = appId => findOnMarket(self, appId)
```

> Nada muito importante também...